### ISSUE TITLE
s there other way to get the Microsoft Edge DevTools Preview without Microsoft store?

### ISSUE THREAD

**Issue by esominion** [OfficeDev/office-js#2498]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

The problem is that it seems the only way to install the tools is Microsoft Store. Unfortunately the Miscrosoft Store is blocked on my machine, I want to know is there any other way to get the Microsoft Edge DevTools Preview tool? Any help would be appreciated, thank you in advance.

**Comment by nkkinariwala** [comment]
@millerds - Is it something you can help with?

**Comment by millerds** [comment]
I am not aware of how to get this or any other store app outside of the store.
